.github/workflows/changelog.yml

```diff
@@ -0,0 +1,31 @@
+# This action requires that any PR targeting the master branch should touch at
+# least one CHANGELOG file. If a CHANGELOG entry is not required, add the "Skip
+# Changelog" label to disable this action.
+
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+    branches:
+      - master
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.pull_request.labels.*.name, 'Skip Changelog')"
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Check for CHANGELOG changes
+      run: |
+        git fetch origin ${{ github.base_ref }} --depth=1
```

Comment:
Does this commit need to be fetched vs just checking it out?

Reply:
Yep, because the checkout action only gives us the latest commit on the feature branch by default. Added a comment.